CruCentralCoast/Profile/Cells/ProfileHeaderView.swift

```diff
@@ -7,18 +7,27 @@
 //
 
 import UIKit
+import FirebaseAuth
 
 class ProfileHeaderView: UITableViewHeaderFooterView {
 
-    @IBOutlet weak var imageLabel: UIImageView!
+    @IBOutlet weak var imageView: UIImageView!
     @IBOutlet weak var nameLabel: UILabel!
-    @IBOutlet weak var fakeBottomOfNavBar: UIView!
     
     override func awakeFromNib() {
         super.awakeFromNib()
-        self.fakeBottomOfNavBar.addBorders(edges: .bottom, color: .navBarLineGray, thickness: 0.5)
+        
+        self.imageView.layer.cornerRadius = self.imageView.frame.width / 2
+        self.imageView.clipsToBounds = true
     }
     
-    
-    
+    func configure(with user: User?) {
+        if user == nil {
+            self.imageView.image = #imageLiteral(resourceName: "profile_icon")
+            self.nameLabel.text = "First Last"
```

Comment:
if they're not signed in, showing "First Last" probably doesn't make sense. we could more thought into this, but what about something like "Guest" or something else indicating they're not logged in

Reply:
Yeah, "Guest" definitely sounds better